### PR TITLE
[BUGFIX] Remove double "read"

### PR DESCRIPTION
**Pull request by simonschaufi**

### CODE REVIEW

File: Documentation/ColumnsConfig/Type/None/Index.rst [hunk 1]
```diff
@@ -17,7 +17,7 @@ Characteristics of `none`:
 
 * The DataHandler discards values send for type none and never persists or updates them in the database.
 * Type none is the only type that does **not** necessarily need a database field.
-* Type none fields does have a default renderType in FormEngine that displays the value as read read only
+* Type none fields does have a default renderType in FormEngine that displays the value as read only
```
Comment (mbrodala) [inline Documentation/ColumnsConfig/Type/None/Index.rst:20]:
```suggestion
* Type none fields does have a default renderType in FormEngine that displays the value as readonly
```

**Review by mbrodala — CHANGES_REQUESTED** [review]

File: Documentation/ColumnsConfig/Type/None/Index.rst (hunk 1 above)
Comment (simonschaufi) [reply Documentation/ColumnsConfig/Type/None/Index.rst:20]: @mbrodala There is another occurrence of it in "If no database field exists for none fields, an empty read only input field is rendered by default." Which then should also be adjusted, or?

Comment (mbrodala) [reply Documentation/ColumnsConfig/Type/None/Index.rst:20]: Yes, please adjust all occurences.

**Review by linawolf — APPROVED** [review]

**Comment by linawolf** [comment]
@simonschaufi If you need your changes backportet just write Releases: master, 10.4, 9.5 no need to make a separate PR for each version

**Pull request merged**